etc/generate-ssl-certs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Uses mkcert to generate self-signed SSL/TLS certificates to allow Traefik to manage HTTPS
+# connections for a local deployment for Rosalution. Automatically installs the generated certificate
+# with the Chrome browser trust to let the browser know the certificate is valid and secure when
+# accessing local.rosalutin.cgds/rosalution.
+
+# Also creates the certificate directory path, default is ./etc/.certificates
+
+# ./generate-ssl-certs.sh <hostname> <certificate path>
+# ./generate-ssl-certs.sh local.rosalution.cgds ./etc/.certificates
+
+
+HOSTNAME=$1
+CERT_PATH=$2
+
+mkdir "$CERT_PATH"
```

Comment:
Recommend we use the "-p" option in mkdir so that it creates the full file path if it only partially exists and does not fail.

Reply:
Apologies, I thought the command was using `-p`. This has been updated.